safety_limiter/test/src/test_safety_limiter.cpp

```diff
@@ -609,6 +609,86 @@ TEST_F(SafetyLimiterTest, SafetyLimitMaxVelocitiesValues)
   }
 }
 
+TEST_F(SafetyLimiterTest, SafetyLimitOmniDirectional)
+{
+  ros::Rate wait(20.0);
+  const double vel = 1.5;
+  const double threshold_angle = std::atan2(1.0, 0.1);
+  for (double angle = -M_PI; angle < M_PI; angle += M_PI / 8)
+  {
+    for (int i = 0; i < 10 && ros::ok(); ++i)
+    {
+      publishSinglePointPointcloud2(2.0, 0, 0, "base_link", ros::Time::now());
+      publishWatchdogReset();
+      publishTwist(0, 0);
+      wait.sleep();
+      ros::spinOnce();
+    }
+
+    double obstacle_x = 0.5 * std::cos(angle);
+    double obstacle_y = 0.5 * std::sin(angle);
+    if ((M_PI - threshold_angle) < std::abs(angle))
+    {
+      // Moving backward
+      obstacle_x += -2.0;
+    }
+    else if (threshold_angle < std::abs(angle))
+    {
+      if (angle > 0)
+      {
+        // Moving left
+        obstacle_x += -1.0;
+        obstacle_y += 0.1;
+      }
+      else
+      {
+        // Moving right
+        obstacle_x += -1.0;
+        obstacle_y += -0.1;
+      }
+    }
+    // Otherwise moving forward.
+
+    // 1.0 m/ss, obstacle at 0.5 m: limited to 1.0 m/s
```

Comment:
```suggestion
    // 1.5 m/ss, obstacle at 0.5 m: limited to 1.0 m/s
```
https://github.com/at-wat/neonavigation/pull/646/files#diff-af1b74605ce195b79787573d2e720a9f7d3ae2721510085f2f39938157d59da3R615

Reply:
Done